gtsfm/bundle/bundle_adjustment.py

```diff
@@ -28,6 +41,78 @@ class BundleAdjustmentOptimizer:
     This class refines global pose estimates and intrinsics of cameras, and also refines 3D point cloud structure given
     tracks from triangulation."""
 
+    def __init__(self, shared_calib: bool = False):
+        """Initializes the parameters for bundle adjustment module.
+
+        Args:
+            shared_calib (optional): Flag to enable shared calibration across
```

Comment:
shared_calib should go in the config i believe. COLMAP allows this as a config parameter

Reply:
oh nevermind, i saw you added that